src/schema/schema.graphql

```diff
@@ -280,10 +280,36 @@ type AuthResponse {
 	skin: [Skin]
 }
 
+type ScratchCode{
+	ScratchCode: String
+}
+
+type EnableTwoFactorAuthResponse {
+	secret: [Secret]
+	scratchCodes: [ScratchCode]
+	authToken: [TwoFactorAuthToken]
+}
+
 type ProfileImageChangeResponse {
 	itemId: ID
 }
 
+type TwoFactorAuthToken {
```

Comment:
we already have authtoken type
`type AuthToken`
please reuse it here

Reply:
done